### PR TITLE
docs: Add installation instruction for local-gadget

### PR DESCRIPTION
**Pull request by blanquicet**
# Add installation instruction for local-gadget

It also updates the index in the installation guide.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -159,3 +161,14 @@ deployed on the cluster to be the exact same version. Even if this is
 possible that different versions work well together, we don't provide
 any guarantee in those cases. We'll visit this policy again once we
 approach to the v1.0 release.
+
+## Installing local-gadget
+
```
Comment (mqasimsarfraz) [inline docs/install.md:166]: How about adding the steps to download and install from the release assets?

File: docs/install.md [hunk 2]
```diff
@@ -159,3 +161,14 @@ deployed on the cluster to be the exact same version. Even if this is
...
+The [`local-gadget`](local-gadget.md) tool can be built and install
+independently. The result is a single binary (statically linked) that can be
```
Comment (flyth) [inline docs/install.md:168]:
```suggestion
The [`local-gadget`](local-gadget.md) tool can be built and installed
independently. The result is a single binary (statically linked) that can be
```

File: docs/install.md [hunk 3]
```diff
@@ -159,3 +161,14 @@ deployed on the cluster to be the exact same version. Even if this is
...
+copied to a Kubernetes node or any host to trace its containers:
+
+```bash
+$ make local-gadget
+$ sudo cp local-gadget /usr/local/bin/
```
[the remark this answers is not in the document]
Comment (blanquicet) [reply docs/install.md:173]: Yep, I will do it in another PR.

File: docs/local-gadget.md [hunk 4]
```diff
@@ -35,6 +35,11 @@ The architecture of `local-gadget` is described in the main
 - Outside a Kubernetes environment, for observing and debugging standalone
   containers.
 
+## Installation
+
+The instruction to install `local-gadget` are available in the main
+[installation](install.md#installing-local-gadget) guide.
```
[the remark this answers is not in the document]
Comment (blanquicet) [reply docs/local-gadget.md:41]: I agree that it needs to be better organised. Let's discuss it in https://github.com/kinvolk/inspektor-gadget/issues/900.

**Review by mauriciovasquezbernal — APPROVED** [review]
LGMT!

**Pull request merged**